builder/builder.go

```diff
@@ -1838,6 +1838,12 @@ func (b *Builder) AddRPMList(rpms []string) error {
 		if _, err := os.Stat(repoPath); err == nil {
 			continue
 		}
+		// Remove source RPMs because they should not be added to mixes
+		if strings.Contains(rpm, ".src.rpm") {
+			if err := os.RemoveAll(filepath.Join(b.Config.Mixer.LocalRPMDir, rpm)); err != nil {
+				return errors.Wrapf(err, "Failed to remove %s, your mix will not generate properly with source RPMs included.", rpm)
+			}
```

Comment:
No warning when we are removing files they put there?

Reply:
I will add one 👍